### PR TITLE
Formatted README

### PR DESCRIPTION
**Pull request by MEME-MAN234**
I have updated the README.md and added CONTRIBUTING.md

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/79)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,44 +6,116 @@
...
 
-Technical Setup
-===============
+<a name="desc"></a>
 
-#### Run docker image to start instance.
+## 1. Description
 
-```bash
+This is a part of project XCoV19 which aims to to build a software that guides patients to the nearest hospital, while modeling the spread of the virus in order to better anticipate hospital needs. [Here](https://www.covidsos.net/website/home.html) is the official page.
```
Comment (codecakes) [inline README.md:23]: two "to"'s typo.

Comment (codecakes) [inline README.md:23]: https://www.covidsos.net is enough for the link, since subroutes may change in future.

File: README.md [hunk 2]
```diff
@@ -6,44 +6,116 @@
...
+
+## 7. Credits
+
+(IN NO SPECIFIC ORDER)
+
+### [ codecakes ](https://github.com/codecakes)
+
+### [ GeekySankar ](https://github.com/GeekySankar)
+
+### [ akshayparmar90 ](https://github.com/akshayparmar90)
```
Comment (codecakes) [inline README.md:121]: @MEME-MAN234 add yourself. nicely done!

**Review by codecakes — CHANGES_REQUESTED** [review]
It's a very nice PR. some minor changes but overall nicely done!

**Review by codecakes — COMMENTED** [review]
ill approve once minor edits are done. you may then merge;

**Comment by MEME-MAN234** [comment]
I have resolved them!

**Comment by MEME-MAN234** [comment]
@codecakes Please check my request now!

**Review by codecakes — APPROVED** [review]

**Comment by MEME-MAN234** [comment]
Can you merge this?

**Comment by MEME-MAN234** [comment]
@codecakes Do you want me to add something else?

**Comment by codecakes** [comment]
@MEME-MAN234 go ahead, you should be able to merge this

**Comment by MEME-MAN234** [comment]
@codecakes No I am unable to this since I don't have write access to this repo

**Comment by MEME-MAN234** [comment]
@codecakes Can you merge this? Since I don't have write access to this repo I can't do this.

**Comment by codecakes** [comment]
> @codecakes Can you merge this? Since I don't have write access to this repo I can't do this.

go ahead, you should be able to now

**Comment by MEME-MAN234** [comment]
@codecakes check the screenshot.
x

**Pull request merged**